BootloaderCommonPkg/Library/IppCryptoFipsLib/IppCryptoFipsLibM7.inf

```diff
@@ -0,0 +1,146 @@
+## @file
```

Comment:
Better add library descript used only when hardware support SSE3?

Reply:
I've added a readme file with some explanation of the different optimization levels.